.github/actions/iotedgedev/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo -E iotedgedev genconfig -f $2/$1 -P $3  --fail-on-validation-error
```

Comment:
Can we add a description of what this file does and why, at the beginning of the file itself?